test-case-reporting/test/fixtures/sample-travis-build.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
For something this small, I'd just hard-code it in the test. Fixtures are better used for larger payloads, such as the full request Travis might make to the app

Reply:
Done.